modules/gdscript/language_server/gdscript_workspace.cpp

```diff
@@ -520,8 +520,29 @@ void GDScriptWorkspace::completion(const lsp::CompletionParams &p_params, List<S
 
 	if (const ExtendGDScriptParser *parser = get_parse_result(path)) {
 		Node *owner_scene_node = _get_owner_scene_node(path);
+
+		Array stack;
+		Node *current;
```

Comment:
```
modules/gdscript/language_server/gdscript_workspace.cpp: In member function 'void GDScriptWorkspace::completion(const lsp::CompletionParams&, List<ScriptCodeCompletionOption>*)':
modules/gdscript/language_server/gdscript_workspace.cpp:525:9: error: 'current' may be used uninitialized in this function [-Werror=maybe-uninitialized]
  525 |   Node *current;
      |         ^~~~~~~
```

Probably not true unless `push_back` can fail, but you can initialize to `nullptr` anyway.